android/src/main/java/com/horcrux/svg/SvgViewManager.java

```diff
@@ -315,32 +315,27 @@ public void setNextFocusLeft(SvgView view, int value) {
   }
 
   @Override
-  @ReactProp(name = "borderBottomLeftRadiusAndroid")
-  public void setBorderBottomLeftRadiusAndroid(SvgView view, float value) {
-    super.setBorderBottomLeftRadius(view, value);
+  public void setBorderRadius(SvgView view, double value) {
+    super.setBorderRadius(view, 0, (float) value);
   }
 
   @Override
-  @ReactProp(name = "borderBottomRightRadiusAndroid")
-  public void setBorderBottomRightRadiusAndroid(SvgView view, float value) {
-    super.setBorderBottomRightRadius(view, value);
+  public void setBorderTopLeftRadius(SvgView view, double value) {
+    super.setBorderTopLeftRadius(view, (float) value);
```

Comment:
@WoLewicki, this will call BaseViewManager.setBorderTopLeftRadius().

https://github.com/facebook/react-native/blob/048194849bda980eecf7bb006ca4e71e2d60ff4c/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L629

This will throw, or log errors, when the borderRadius prop is set.

Instead, we want these methods to call ReactViewManager.setBorderRadius(), no?

https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java#L111-L138

That way, React Native will actually assign the borderRadius to the SvgView.

Reply:
Yeah, you are right 🚀